### ISSUE TITLE
x/build: set of first class ports is out of sync with porting policy

### ISSUE THREAD

**Issue by findleyr** [golang/go#44233]
I'm trying to clean up the x/tools build dashboard, starting with first class ports. Per the [porting policy](https://github.com/golang/go/wiki/PortingPolicy) broken builds in first class ports should block releases. However, the set of first class ports listed in our [build dashboard](https://build.golang.org/?repo=golang.org%2fx%2ftools#short) is out of sync with the ports listed in that policy.

Which is authoritative? Can we get them in sync?

CC @golang/release

**Comment by dmitshur** [comment]
> Which is authoritative?

The list in https://golang.org/wiki/PortingPolicy#first-class-ports is authoritative.

> Can we get them in sync?

Yes, we should. There's work in progress but it's was blocked on even more work in #34744, which put on hold during the 1.16 freeze. Issue #27689 tracks this task, so I'll close this issue as a duplicate.

**Comment by findleyr** [comment]
Ah, I missed that issue. Thanks Dmitri.
